src/main/java/com/mongodb/spark/sql/connector/config/WriteConfig.java

```diff
@@ -67,6 +71,41 @@ public String toString() {
     }
   }
 
+  /** The convert json configuration. */
+  public enum ConvertJson {
+    /** No conversion of string types */
+    FALSE("false"),
+    /** Try to parse any string as a json value */
+    ANY("any"),
+    /** Try to parse any string as a json value */
+    OBJECT_OR_ARRAY_ONLY("objectOrArrayOnly");
+
+    private final String value;
+    private static final String TRUE = "true";
+
+    ConvertJson(final String operationType) {
+      this.value = operationType;
+    }
+
+    static ConvertJson fromString(final String jsonType) {
+      for (ConvertJson convertJsonType : ConvertJson.values()) {
+        if (jsonType.equalsIgnoreCase(TRUE)) {
+          LOGGER.warn("{}: 'true' is deprecated. Use: '{}' instead.", CONVERT_JSON_CONFIG, ANY);
```

Comment:
I didn't feel the need to introduce a TRUE - value rather opted just to log that its deprecated.